Dave-mash/Questioner-api
# Ft create user 163048839

### What does this PR do?
Builds the register user endpoints

### Description of Task to be completed?
* Build the user model
* Build the register user endpoint
* Validate all user inputs

### How should this be tested
* Clone this repository
`git clone https://github.com/Dave-mash/Questioner-api.git`
* Navigate into the project directory
`cd Questioner-api`
* activate virtualenv
`source venv/bin/activate`
* Install requirements
`pip install -r requirements.txt`
* Run pytest to see passing tests
`python -m pytest`
* Test the endpoint using postman

### What are the relevant pivotal tracker stories
[163048839](https://www.pivotaltracker.com/story/show/163048839)

screenshot:
run app
![screenshot from 2019-01-10 00-07-17](https://user-images.githubusercontent.com/41325679/50928485-d6d74880-146b-11e9-9740-261777a16398.png)

test app
![passusertest](https://user-images.githubusercontent.com/41325679/50928408-a4c5e680-146b-11e9-9a79-64b8b4ff6483.png)
